test/py/onnx_backend_test.py

```diff
@@ -556,9 +556,89 @@ def disabled_tests_onnx_1_14_0(backend_test):
+    backend_test.exclude(
+        r'test_gridsample_volumetric_bilinear_align_corners_0_cpu')
+    backend_test.exclude(
+        r'test_gridsample_volumetric_bilinear_align_corners_1_cpu')
+    backend_test.exclude(
+        r'test_gridsample_volumetric_nearest_align_corners_0_cpu')
+    backend_test.exclude(
+        r'test_gridsample_volumetric_nearest_align_corners_1_cpu')
+    backend_test.exclude(r'test_group_normalization_epsilon_cpu')
+    backend_test.exclude(r'test_group_normalization_example_cpu')
+    backend_test.exclude(r'test_quantizelinear_int16_cpu')
+    backend_test.exclude(r'test_quantizelinear_uint16_cpu')
+    backend_test.exclude(r'test_qlinearmatmul_2D_int8_float16_cpu')
+    backend_test.exclude(r'test_qlinearmatmul_2D_int8_float32_cpu')
+    backend_test.exclude(r'test_qlinearmatmul_2D_uint8_float16_cpu')
+    backend_test.exclude(r'test_qlinearmatmul_2D_uint8_float32_cpu')
+    backend_test.exclude(r'test_qlinearmatmul_3D_int8_float16_cpu')
+    backend_test.exclude(r'test_qlinearmatmul_3D_int8_float32_cpu')
+    backend_test.exclude(r'test_qlinearmatmul_3D_uint8_float16_cpu')
+    backend_test.exclude(r'test_qlinearmatmul_3D_uint8_float32_cpu')
+    backend_test.exclude(r'test_quantizelinear_blocked_asymmetric_cpu')
+    backend_test.exclude(r'test_quantizelinear_blocked_symmetric_cpu')
+    backend_test.exclude(r'test_reduce_l1_empty_set_cpu')
+    backend_test.exclude(r'test_reduce_l1_empty_set_expanded_cpu')
+    backend_test.exclude(r'test_reduce_l2_empty_set_cpu')
+    backend_test.exclude(r'test_reduce_l2_empty_set_expanded_cpu')
+    backend_test.exclude(r'test_reduce_log_sum_empty_set_cpu')
+    backend_test.exclude(r'test_reduce_log_sum_empty_set_expanded_cpu')
+    backend_test.exclude(r'test_reduce_log_sum_exp_empty_set_cpu')
+    backend_test.exclude(r'test_reduce_log_sum_exp_empty_set_expanded_cpu')
+    backend_test.exclude(r'test_reduce_max_bool_inputs_cpu')
+    backend_test.exclude(r'test_reduce_min_bool_inputs_cpu')
+    # TODO: empty set ReduceOps tests are generating dynamic shapes
+    backend_test.exclude(r'test_reduce_min_empty_set_cpu')
+    backend_test.exclude(r'test_reduce_prod_empty_set_cpu')
+    backend_test.exclude(r'test_reduce_sum_empty_set_cpu')
+    backend_test.exclude(
+        r'test_reduce_sum_empty_set_non_reduced_axis_zero_cpu')
+    backend_test.exclude(r'test_reduce_sum_square_empty_set_cpu')
+    backend_test.exclude(r'test_reduce_sum_square_empty_set_expanded_cpu')
+    # TODO: Pooling tests are failing with shape mismatches, look into it.
+    backend_test.exclude(
+        r'test_averagepool_3d_dilations_large_count_include_pad_is_0_ceil_mode_is_True_cpu')
+    backend_test.exclude(
+        r'test_averagepool_3d_dilations_large_count_include_pad_is_1_ceil_mode_is_True_cpu')
+    backend_test.exclude(r'test_maxpool_2d_ceil_output_size_reduce_by_one_cpu')
+    backend_test.exclude(r'test_maxpool_3d_dilations_use_ref_impl_large_cpu')
```

Comment:
So these ones are failing because `rewrite_pooling` pass is not factoring in `ceil_mode` when computing output shape.

I created an issue to track it : https://github.com/ROCm/AMDMIGraphX/issues/3356